src/lib/github.ts

```diff
@@ -41,4 +43,41 @@ async function squashAndMerge({ repo, prNumber }: ActionPayload): Promise<void>
   });
 }
 
-export { approve, squashAndMerge };
+async function askForReview({ repo, prNumber }: ActionPayload, reviewers: string[], message?: string): Promise<void> {
+  const octokit = getClient();
+
+  const body =
+    "🚧 Manual check needed 🚧\n" +
+    (message ? ":\n**" + message + "**" : ".") +
+    "\n\n" +
+    reviewers.reduce((acc, reviewer) => `${acc}@${reviewer} `, "");
```

Comment:
Example:
<img width="994" alt="image" src="https://user-images.githubusercontent.com/29123047/196216316-1ff5dfb7-31b0-4874-9faa-036db5ea3b9d.png">

(I added more spaces after the message has been posted: `\n\n`)

Reply:
Not wrong per say but:
```
> let reviewers = ["Fenntasy", "Meyclem"]
undefined
> reviewers.reduce((acc, reviewer) => `${acc}@${reviewer} `, "")
'@Fenntasy @Meyclem '
> reviewers.map(reviewer => `@${reviewer}`).join(" ")
'@Fenntasy @Meyclem'
```

Notice the lack of space in the end with the second method 😁

Reply:
Nice!
I added it